CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## Stache 1.1.3
```

Comment:
Let it be 1.2.0 instead (you have added a backwards-compatible feature, and it's semver).